x-pack/plugins/ingest_pipelines/public/application/components/pipeline_processors_editor/components/shared/map_processor_type_to_form.tsx

```diff
@@ -121,69 +124,123 @@ export const mapProcessorTypeToDescriptor: MapProcessorTypeToDescriptor = {
     label: i18n.translate('xpack.ingestPipelines.processors.label.enrich', {
       defaultMessage: 'Enrich',
     }),
+    description: () => (
+      <FormattedMessage
+        id="xpack.ingestPipelines.processors.description.enrich"
+        defaultMessage="Adds enrich data to incoming documents based on an {enrichPolicyLink}."
+        values={{
+          enrichPolicyLink: (
+            <EuiLink external target="_blank" href={esDocUrl + '/ingest-enriching-data.html'}>
+              {'enrich policy'}
+            </EuiLink>
```

Comment:
@jloleysens I'm having trouble getting descriptions with an `EuiLink` to display. I think this is likely because I don't have `esDocUrl` defined here. Any pointers on where I should define it?